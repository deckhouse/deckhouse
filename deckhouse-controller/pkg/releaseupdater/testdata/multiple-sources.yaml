---
apiVersion: deckhouse.io/v1alpha1
kind: DeckhouseRelease
metadata:
  name: v1.50.0
spec:
  version: v1.50.0
  requirements:
    migratedModules: "test-module-1"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  name: source-1
spec:
  registry:
    repo: registry1.example.com/modules
    scheme: HTTPS
status:
  modules:
    - name: other-module
  modulesCount: 1
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  name: source-2
spec:
  registry:
    repo: registry2.example.com/modules
    scheme: HTTPS
status:
  modules:
    - name: test-module-1
  modulesCount: 1