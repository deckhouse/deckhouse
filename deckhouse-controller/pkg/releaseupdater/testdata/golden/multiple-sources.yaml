---
apiVersion: deckhouse.io/v1alpha1
approved: false
kind: DeckhouseRelease
metadata:
  creationTimestamp: null
  name: v1.50.0
  resourceVersion: "999"
spec:
  requirements:
    migratedModules: test-module-1
  version: v1.50.0
status:
  approved: false
  message: ""
  transitionTime: null
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  creationTimestamp: null
  name: source-1
  resourceVersion: "999"
spec:
  registry:
    ca: ""
    dockerCfg: ""
    repo: registry1.example.com/modules
    scheme: HTTPS
status:
  message: ""
  modules:
  - name: other-module
  modulesCount: 1
  phase: ""
  syncTime: null
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  creationTimestamp: null
  name: source-2
  resourceVersion: "999"
spec:
  registry:
    ca: ""
    dockerCfg: ""
    repo: registry2.example.com/modules
    scheme: HTTPS
status:
  message: ""
  modules:
  - name: test-module-1
  modulesCount: 1
  phase: ""
  syncTime: null
---
data:
  cluster-configuration.yaml: a3ViZXJuZXRlc1ZlcnNpb246ICIxLjI5Ig==
metadata:
  creationTimestamp: null
  name: d8-cluster-configuration
  namespace: kube-system
  resourceVersion: "1"
